### PR TITLE
Songs Searched for Globally Component

### PR DESCRIPTION
**Pull request by kernelkay**
### Added "Songs Searched for Globally" Component and MusicPlayer component
- Can toggle number of songs shown per slide (set to 4)
- has autoplay and slide animation
- Added sample song data taken from Spotify API
- Added MusicPlayer component to play song previews

#### With MusicPlayer Component played
Screen Shot 2023-06-04 at 2 55 15 PM

#### Hovered
Screen Shot 2023-06-04 at 3 00 29 PM

#### With wrapping
Screen Shot 2023-06-04 at 3 00 39 PM

### CODE REVIEW

File: src/components/GloballySearched.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            return <SongGroup key={i} songs={group} />;
+          })}
+          {}
+        </Carousel>
+      </div>
+    </div>
+  );
+}
+
+function SongGroup({ songs }) {
```
Comment (Parmvir-S) [inline src/components/GloballySearched.js:42]: Should this be its own component?

File: src/components/GloballySearched.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  return (
+    <Paper className='carousel-song-group'>
+      {songs.map((song, i) => {
+        return <Song key={i} song={song} />;
+      })}
+    </Paper>
+  );
+}
+
+function Song({ song }) {
```
Comment (Parmvir-S) [inline src/components/GloballySearched.js:52]: Same thing here

File: src/components/GloballySearched.js (hunk 1 above)
Comment (kernelkay) [reply src/components/GloballySearched.js:42]: Made it isolated as we likely won't be reusing this type of component. It's specific to the carousel component.

File: src/components/GloballySearched.js (hunk 2 above)
Comment (kernelkay) [reply src/components/GloballySearched.js:52]: Same for this one, this component would only likely fit for the carousel component, although we can extract it if needed.

File: src/components/GloballySearched.js (hunk 1 above)
Comment (Parmvir-S) [reply src/components/GloballySearched.js:42]: Ok that makes sense. Thanks for clarifying!

**Review by rrichc — APPROVED** [review]
I think this looks good, gonna merge this in first then Parm's

**Pull request merged**